### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by ridhoassuryadi**
Lohaaaa!!, lets contribute to indonesian developer

### CODE REVIEW

**Comment by muslih** [comment]
Hi mang @muhammadridho can you also add background why did you initiate to create this repo? :bowing_man:

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # indonesia-holiday
-🇮🇩 Check Indonesian holiday time
+🇮🇩 Check Indonesian holiday time
```
Comment (muslih) [inline README.md:2]: I am a noob. Would you mind to add steps on how to set up, dependencies, etc :baby:

File: __tests__/index_test.re [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* Reason */
+open Jest;
+
+let () =
+  describe("Expect", () =>
+    Expect.(test("toBe", () =>
+              expect(1 + 2) |> toBe(3)
```
Comment (muslih) [inline __tests__/index_test.re:7]: an intended space? :thinking:

File: README.md (hunk 1 above)
Comment (ridhoassuryadi) [reply README.md:2]: at the first time this is just init, so there will be step to add `readme.md`, 😁

File: __tests__/index_test.re (hunk 2 above)
Comment (ridhoassuryadi) [reply __tests__/index_test.re:7]: yo

**Comment by ridhoassuryadi** [comment]
> Hi mang @muhammadridho can you also add background why did you initiate to create this repo? 🙇

hello this will be an opportunity for us as an active community in contributing to open source, I want us as an active community.

**Comment by muslih** [comment]
> hello this will be an opportunity for us as an active community in contributing to open source, I want us as an active community.

That would be nice if we put it and the purpose of the repo on readme :heart_eyes:  @muhammadridho

File: __tests__/index_test.re (hunk 2 above)
Comment (muslih) [reply __tests__/index_test.re:7]: @muhammadridho  is it okay?

Comment (ridhoassuryadi) [reply __tests__/index_test.re:7]: i think we need prettier, it is because reformatter from reasonml

**Comment by ridhoassuryadi** [comment]
> > hello this will be an opportunity for us as an active community in contributing to open source, I want us as an active community.
> 
> That would be nice if we put it and the purpose of the repo on readme 😍 @muhammadridho

hahaha, slowly. its just first initial

**Review by muslih — APPROVED** [review]

**Pull request merged**